### PR TITLE
Add githubstudent's pin to the map!

### PR DESCRIPTION
**Pull request by julianWEBERSTATE**
Add Ogden to the Map!

closes #940

### CODE REVIEW

**Comment by SaltMiner** [comment]
Wonder why the CI build failed?

File: julianWEBERSTATE.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: julianWEBERSTATE
+latitude: 41.190927
+longitude: -111.941864
+---
```
Comment (hectorsector) [inline julianWEBERSTATE.json]: @julianWEBERSTATE remove this file

File: _pins/gitHubStudent-Ogden.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: Julian
+Latitude: 41.190927
+Longitude: -111.946092
+---
```
Comment (hectorsector) [inline _pins/gitHubStudent-Ogden.json]: @julianWEBERSTATE remove this file as well

File: _pins/julianWEBERSTATE.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: julianWEBERSTATE
+latitude: 41.190927
+longitude: -111.941864
+---
```
Comment (hectorsector) [inline _pins/julianWEBERSTATE.json]: @julianWEBERSTATE this file looks good!

File: julianWEBERSTATE.json (hunk 1 above)
Comment (julianWEBERSTATE) [reply julianWEBERSTATE.json]: Add file back

**Pull request merged**